pkg/apis/workflow/v1alpha1/task_set_types.go

```diff
@@ -1,9 +1,11 @@
 package v1alpha1
 
-import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
 
 // +genclient
-// +kubebuilder:subresource:status
```

Comment:
don't you still need this line?

Reply:
we don't need it. code is not using `UpdateStatus`